queries/editorconfig/highlights.scm

```diff
@@ -1,8 +1,7 @@
 (comment) @comment @spell
 
-; consistency with toml and ini
 (section
-  (section_name) @type)
+  (section_name) @markup.heading)
```

Comment:
Again, I don't think these are actual headings -- they are globs and should be highlighted as such: `@string.special.path` or `@regexp`.

Reply:
Changed to `@string.special.path`

Reply:
@gpanders